src/events/Windows.cpp

```diff
@@ -198,6 +198,46 @@ void Events::listener_mapWindow(void* owner, void* data) {
             overridingNoMaximize = true;
         } else if (r.szRule == "stayfocused") {
             PWINDOW->m_bStayFocused = true;
+        } else if (r.szRule.find("group") == 0) {
+            if (PWINDOW->m_eGroupRules & GROUP_OVERRIDE)
+                continue;
+
+            // `group` is a shorthand of `group set`
+            if (removeBeginEndSpacesTabs(r.szRule) == "group") {
+                PWINDOW->m_eGroupRules |= GROUP_SET;
+                continue;
+            }
+
+            CVarList vars(r.szRule, 0, 's');
+            bool     hasDeny = false;
+
+            for (auto const& v : vars) {
+                if (v == "group")
+                    continue;
+                if (v == "set")
+                    PWINDOW->m_eGroupRules |= GROUP_SET;
+                else if (v == "always")
+                    PWINDOW->m_eGroupRules |= GROUP_ALWAYS;
+                else if (v == "locked")
+                    PWINDOW->m_eGroupRules |= GROUP_LOCK;
+                else if (v == "deny")
+                    hasDeny = true;
+                else if (hasDeny && v == "entry")
+                    PWINDOW->m_eGroupRules |= GROUP_DENY_ENTRY;
+                else if (hasDeny && v == "exit")
+                    PWINDOW->m_eGroupRules |= GROUP_DENY_EXIT;
+                else if (v == "override")
+                    // Clear existing rules
+                    PWINDOW->m_eGroupRules = GROUP_OVERRIDE;
+                else if (v == "unset") {
```

Comment:
`group unset` is shorthand for `group override`, but also immediately stops processing other group rules.